lib/adapter/codecept.js

```diff
@@ -76,11 +112,15 @@ function CodeceptReporter(config) {
     if (!global.testomatioDataStore) global.testomatioDataStore = {};
     // reset steps
     global.testomatioDataStore.steps = [];
+
+    logger.setContext('BEFORE_TEST_' + test.fullTitle());
```

Comment:
```suggestion
    logger.setContext(test.fullTitle());
```
This should be the part of test.

Here is the difference:

test.before = async
test.started = sync

So test.before starts a test promise which when resolved will fire test.started

So test before is a test anyway.